torch/csrc/jit/script/compiler.cpp

```diff
@@ -2045,7 +2057,8 @@ struct to_ir {
         throw ErrorReport(loc) << "Only one argument to __new__ allowed";
       }
       return classNew->createObject(
-          apply.range(), method, Var(apply.inputs()[0]).name().name());;
+          apply.range(), method, Var(apply.inputs()[0]).name().name());
+      ;
```

Comment:
?

Reply:
Formatting change that i didn't look closely in, will remove extra `;`